pages/about.mdx

```diff
@@ -26,7 +26,21 @@ source project has a license that determines the way people can use, distribute,
 Join the Fiveable Discord server at [discord.gg/thinkfiveable](https://discord.gg/thinkfiveable) and check out
 the #open-source channel which serves as the main discussion forum for this initiative. The GitHub repository
 can be found [here](https://github.com/ThinkFiveable/open). We are currently accepting submissions for open
-source projects
-[here](https://docs.google.com/forms/d/e/1FAIpQLSeMuqyb8Tdf3Z2oY0kkFN7frYQT9RX97X09diiCW6FtoFZ1rQ/viewform).
+source projects through pull requests.
 
-Don't know where to start? Check out our [Git tutorials](/docs/git).
+## How do I submit my project?
+
+Head over to our [GitHub repository](https://github.com/thinkfiveable/open) and navigate to the file
+`PROJECTS.JSON` located in the directory `data`. Add the information for your project in the following format:
+
+```json
+     {
+        "repoName": "Name of your repo here",
+        "repoOwner": "GitHub username here",
+        "projectType": "Type of project",
+        "projectDescription": "Describe your project here."
```

Comment:
Can we add something about people being able to add logos?

Reply:
👍